### ISSUE TITLE
AMReX_base_mod.F90 is not installed by CMake

### ISSUE THREAD

**Issue by certik** [AMReX-Codes/amrex#63]
Here is how I installed & tested the latest master (3724a070adf93b46c4170eccb6d295e466abaed8) of AMReX:
```
module load cmake/3.6.2 gcc/5.3.0 openmpi/1.10.5
cd $HOME/repos/amrex
cmake -DCMAKE_INSTALL_PREFIX=inst .
make -j16
make install
cd MiniApps/AMR_Adv_Diff_F90
mpicxx -I$HOME/repos/amrex/inst/include write_plotfile.f90 prob.f90 compute_flux.f90 init_phi.f90 update_phi.f90 advance.f90 main.f90 -L$HOME/repos/amrex/inst/lib -lfboxlib -lcboxlib -lmpi_usempif08
mpirun -n 16 ./a.out
```
And that compiles and runs in parallel.

However, this did not install the `Src/F_Interfaces/Base/AMReX_base_mod.F90` module into `$HOME/repos/amrex/inst/include`. In fact, by looking into the `Src/F_Interfaces/Base/` directory, it seems that it is missing the CMake build system completely.

Is the file `AMReX_base_mod.F90` supposed to be installed by CMake, and if not, why not? What should be the proper fix for this?

**Comment by certik** [comment]
Note: I have been following the https://github.com/AMReX-Codes/amrex/blob/3724a070adf93b46c4170eccb6d295e466abaed8/INSTALL file. The way 1) failed for me due to #65, the way 2) failed due to #64 and the way 3) failed due to this issue above.

**Comment by WeiqunZhang** [comment]
`MiniApps/AMR_Adv_Diff_F90` uses Fortran BoxLib and is no longer supported.  I will remove the example.

**Comment by certik** [comment]
@WeiqunZhang thank you. However, even if you remove the `MiniApps/AMR_Adv_Diff_F90` example, that will not fix the problem that `Src/F_Interfaces/Base/AMReX_base_mod.F90` is not installed. Is the `Src/F_Interfaces/Base/` part of the old Fortran BoxLib, or the new AMReX?

**Comment by WeiqunZhang** [comment]
Yes there is still the issue of cmake support for Fortran interface.  If you require it, please open a new issue.

**Comment by certik** [comment]
@WeiqunZhang thanks, I have reported it as #69.
